src/utils/applyStyles/index.js

```diff
@@ -0,0 +1,2 @@
+export { default } from './applyStyles'
+export * from './applyStyles'
```

Comment:
What is the benefit for this approach? I don't see anything wrong with:
```js
// some-usage-file.js
import applyStyles from './applyStyles'
```

Reply:
You can still import it like that, that's the good thing about it.
The rest of your project doesn't change because of this change the import paths stay the same.